.vscode/settings.json

```diff
@@ -2,20 +2,22 @@
 	"deno.enablePaths": ["scripts/deno"],
 	"deno.lint": true,
 	"deno.unstable": false,
-    "eslint.validate": ["typescript", "typescriptreact"],
-    "editor.codeActionsOnSave": {
-        "source.fixAll.eslint": true
-    },
-    "eslint.workingDirectories": [
-        "./dotcom-rendering"
-	],
-    "files.watcherExclude": {
-        "**/target": true
-    },
-    "workbench.colorCustomizations": {
-        "titleBar.activeBackground": "#648FFF",
-        "titleBar.activeForeground": "#000000"
+	"eslint.validate": ["typescript", "typescriptreact"],
+	"editor.codeActionsOnSave": {
+		"source.fixAll.eslint": true
+	},
+	"eslint.workingDirectories": ["./dotcom-rendering"],
+	"files.watcherExclude": {
+		"**/target": true
 	},
-    "typescript.tsdk": "node_modules/typescript/lib",
-    "gitlens.advanced.blame.customArguments": ["--ignore-revs-file", ".git-blame-ignore-revs"]
+	"workbench.colorCustomizations": {
+		"titleBar.activeBackground": "#648FFF",
+		"titleBar.activeForeground": "#000000"
+	},
+	"typescript.tsdk": "node_modules/typescript/lib",
+	"gitlens.advanced.blame.customArguments": [
+		"--ignore-revs-file",
+		".git-blame-ignore-revs"
+	],
+	"deno.enable": true
```

Comment:
My understanding was that this would enable the deno ls for all files inn the directory, which seems like something we might not want here?

Reply:
Yea I'm not sure why my editor changed this file ... Will be undoing shortly

Reply:
ah yeah fair enough

also -- sorry I hadn't clocked that this was a WIP pr still. I'll review when it's done?

having this action seems like a great idea to me!